exporters/tokens/src/generators/stylesGenerator.ts

```diff
@@ -58,22 +59,27 @@ export const tokenToStyleByType = (
 
   if (hasTokenType(TokenType.color)) {
     const colorToken = token as ColorToken;
-    const name = tokenVariableName(colorToken, tokenGroups, withParent);
-    let value = CSSHelper.colorTokenValueToCSS(colorToken.value, mappedTokens, {
-      allowReferences: true,
-      decimals: 3,
-      colorFormat: ColorFormat.hex8,
-      tokenToVariableRef: () => '',
-    });
-    value = normalizeColor(value);
-    value = handleSpecialCase(name, value);
+    const name = tokenVariableName(colorToken, tokenGroups, hasParentPrefix);
 
-    return formatTokenStyleByOutput(name, value, hasJsOutput);
+    if (hasMixin) {
```

Comment:
**polish (if-minor):** Shouldn't this be scoped in a function?

I am thinking about a refactoring of the file and the structure. In a way there can be a switch that only calls the functions:

```typescript
switch (getTokenType(token)) {
  case TokenType.dimension:
    return generateDimensionTokenStyle(...);
  case ...
}

```